lib/entities/webhook.ts

```diff
@@ -28,33 +28,36 @@ export type WebhookProps = {
   /**
    * Username for basic http auth
    */
-  httpBasicUsername: string
+  httpBasicUsername: string | null
 
   /**
    * Password for basic http auth
    */
-  httpBasicPassword: string
+  httpBasicPassword: string | null
 
   /**
    * Headers that should be appended to the webhook request
    */
-  headers: {
-    [key: string]: string
-  }
+  headers: Array<{ key: string; value: unknown; secret?: boolean }>
```

Comment:
I'm always uncertain with this, but aren't header values always strings? or, get transformed to string?

Reply:
Only ASCII strings are accepted for both key and value